### DISCUSSION TITLE
Category page URL replace with ?p=-Infinity

### DISCUSSION THREAD

**Discussion by web01bharat** [breezefront/community#157] [1. Help]
Hi Support,

We are facing an issue with the Category page URL after going to page-2 it changes with ?p=-Infinity.

As we click to page -2 pagination on the category page it changes ?p-2 to ?p=-Infinity.

Please let us know if we have a solution for this.

Thanks
Bharat

**Comment by vovayatsyuk** [comment]
This is not a default Breeze functionality.

Do you use Swissup_Ajaxlayerednavigation? Which version if "yes"? What module do you use if "no"?

**Reply by web01bharat** [reply]
Hi Vovayatsyuk,

Yes i am using Swissup_Ajaxlayerednavigation ver - 1.4.10

thanks

**Reply by vovayatsyuk** [reply]
Hmm, the same issue was fixed since 1.4.0. 

Can you please open `swissup/module-ajaxlayerednavigation/view/frontend/web/js/navigation/page-detector.js` and check if it has the folllowing code at line 60:

```
if (pageNembers.length > 0) {
    ...
}
```

If you have these lines in your file, try to cleanup the static content folder and redeploy it again.
